javascript/linkedList/README.md

```diff
@@ -30,3 +30,4 @@
 + Sara Strasner and Tina Myers
 
 + ![LL Insert](/assets/insertion.png)
```

Comment:
This line should be:

`![LL Insert](./assets/insertion.png)`

Reply:
Thank you! All fixed